core/components/organisms/table/Header.tsx

```diff
@@ -20,6 +20,7 @@ export interface ExternalHeaderProps {
   dynamicColumn?: boolean;
   allowSelectAll?: boolean;
   customSelectionLabel?: string;
+  tableActionRenderer?: (data: Data) => React.ReactNode;
```

Comment:
Please change this prop name. Specifying `table` in the name is redundant as all these are props of table component only.